doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -41,22 +41,22 @@ Synopsis
 Options
 ~~~~~~~
 
-  -f, --format <html|xml> Specifies whether to generate an HTML or XML
-                          output file. The default output `format` and the 
-                          default output `specdocformat` is got from the
-                          output file extension.
-                          :file:`*.html`    -> `-f HTML -s HTML`
-                          :file:`*.xml`     -> `-f XML  -s RAW`
-                          :file:`*.libspec` -> `-f XML  -s HTML`
-   -s --specdocformat <raw|html>
+  -f, --format <html|xml>
+                          Specifies whether to generate an HTML, XML or
+                          JSON output file. The default output `format`
+                          and the default output `specdocformat` is fetched
+                          from the output file extension.
+                          :file:`*.html` -> `-f HTML -s HTML`,
+                          :file:`*.xml` -> `-f XML -s RAW`,
+                          :file:`*.json` -> `-f JSON -s HTML` and
+                          :file:`*.libspec` -> `-f XML -s HTML`
+  -s, --specdocformat <raw|html>
                           Specifies for XML outputs whether the keyword
```

Comment:
Not only XML anymore. I'd use something like "XML and JSON spec files" to use the word spec also in the documentation and not only in the option name.